viewer/packages/data-providers/src/image-360-data-providers/cdm/transformCdmAnnotations.ts

```diff
@@ -21,11 +22,15 @@ export function transformAnnotations(
     const connectedImage = image360Map[annotation.endNode.externalId][annotation.endNode.space];
 
     const properties = annotation.properties['cdf_cdm']['Cognite360ImageAnnotation/v1'];
+    assert(properties.polygon, 'Polygon must be defined in annotation properties');
```

Comment:
Hmm - can we gracefully fail here somehow? Not sure whether we handle throws like this in a good way

Reply:
We now filter away annotations with a undefined polygon, fixed in [4223fcb](https://github.com/cognitedata/reveal/pull/5277/commits/4223fcb2b8f3d58b268f88f15e9fa87cb7134864)